### PR TITLE
Update kernel version to 5.2.0-m3

### PR DESCRIPTION
**Pull request by ashensw**
Resolves https://github.com/wso2/carbon-jndi/issues/34

### CODE REVIEW

File: tests/test-distribution/pom.xml [hunk 1]
```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 WSO2, Inc. (http://wso2.com)
```
Comment (ChanakaCooray) [inline tests/test-distribution/pom.xml]: Can you update the LICENSE?

Comment (ChanakaCooray) [inline tests/test-distribution/pom.xml]: Can you update the LICENSE?

File: tests/test-distribution/src/assembly/bin.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!--
+ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ChanakaCooray) [inline tests/test-distribution/src/assembly/bin.xml]: Can you update the LICENSE?

**Review by ChanakaCooray — CHANGES_REQUESTED** [review]

File: tests/test-distribution/pom.xml (hunk 1 above)
Comment (ashensw) [reply tests/test-distribution/pom.xml]: done.

File: tests/test-distribution/src/assembly/bin.xml (hunk 2 above)
Comment (ashensw) [reply tests/test-distribution/src/assembly/bin.xml]: done.

**Review by kasunbg — APPROVED** [review]

File: tests/test-distribution/pom.xml [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+                                </artifactItem>
+                            </artifactItems>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.wso2.carbon.maven</groupId>
+                <artifactId>carbon-feature-plugin</artifactId>
+                <version>${carbon.feature.plugin.version}</version>
```
Comment (Niranjan-K) [inline tests/test-distribution/pom.xml]: plugin version should be mentioned in the root pom

**Review by Niranjan-K — CHANGES_REQUESTED** [review]

File: tests/test-distribution/pom.xml (hunk 3 above)
Comment (ashensw) [reply tests/test-distribution/pom.xml]: removed it.

**Review by ChanakaCooray — APPROVED** [review]

File: tests/test-distribution/pom.xml (hunk 3 above)
Comment (kasunbg) [reply tests/test-distribution/pom.xml]: Thanks ashen

**Pull request merged**